website/screens/components/alert/specs/AlertSpecsPage.tsx

```diff
@@ -0,0 +1,541 @@
+import Image from "next/image";
+import AlertSpecsImage from "./images/alert_specs.png";
+import { DxcLink, DxcList, DxcStack, DxcTable, DxcText } from "@dxc-technology/halstack-react";
+import Code from "../../../common/Code";
+import DocFooter from "../../../common/DocFooter";
+import HeadingLink from "../../../common/HeadingLink";
+import Figure from "../../../common/Figure";
+import AlertAnatomyImage from "./images/alert_anatomy.png";
+import AlertStatesImage from "./images/alert_states.png";
+
+const ButtonSpecs = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
```

Comment:
That was also my first thought but there was no text under the image in the original md.

Reply:
I know there wasn't text with the image in the original doc, but after seeing other examples from our partners, I think we should do it like that because of consistency. Again, I'm not fully sure about this.. 😢